### DISCUSSION TITLE
Load SFC component and inject custom data (like HMR)

### DISCUSSION THREAD

**Discussion by akahanek** [FranckFreiburger/vue3-sfc-loader#116] [General]
Hi @FranckFreiburger ,
Is it possible to reload a custom SFC component and preserve its data? I would like to mimic Hot Module Reload:
1. Load component for the first time.
2. Modify the component data through UI
3. Make some changes to the component source.
4. Reload the component and inject the data from the previous instance of the component.

Thanks, Ales

**Comment by tve** [comment]
I would also appreciate any pointers about how to accomplish this!
